packages/conf-gnomecanvas/conf-gnomecanvas.2/opam

```diff
@@ -9,12 +9,18 @@ depexts: [
   ["libgnomecanvas2-dev"] {os-family = "ubuntu"}
   ["libgnomecanvas-devel"] {os-family = "fedora" | os-family = "rhel"}
   ["libgnomecanvas"] {os = "macos" & os-distribution = "homebrew"}
-  ["libgnomecanvas-dev@testing"] {os-family = "alpine"}
+  ["libgnomecanvas-dev@testing" "libart-lgpl-dev"] {os-family = "alpine"}
```

Comment:
Sorry for the long lag. Can we move this change to a separate patched `conf-gnomecanvas.2-1`?